gns3server/controller/project.py

```diff
@@ -1237,3 +1251,62 @@ def __json__(self):
 
     def __repr__(self):
         return "<gns3server.controller.Project {} {}>".format(self._name, self._id)
+
+    async def _duplicate_fast(self, name=None, location=None, reset_mac_addresses=True):
+        # remote replication is not supported
+        if not sys.platform.startswith("linux") and not sys.platform.startswith("win"):
+            return None
+        for compute in self.computes:
+            if compute.id != "local":
+                log.warning("Duplicate fast not support remote compute: '{}'".format(compute.id))
+                return None
+        # work dir
+        p_work = pathlib.Path(location or self.path).parent.absolute()
+        t0 = time.time()
+        new_project_id = str(uuid.uuid4())
+        new_project_path = p_work.joinpath(new_project_id)
+        # copy dir
+        await wait_run_in_executor(shutil.copytree, self.path, new_project_path.as_posix())
+        log.info("[FAST] Copy project: {} to: '{}', cost={}s".format(self.path, new_project_path, time.time() - t0))
+        topology = json.loads(new_project_path.joinpath('{}.gns3'.format(self.name)).read_bytes())
+        project_name = name or topology["name"]
+        # If the project name is already used we generate a new one
+        project_name = self.controller.get_free_project_name(project_name)
+        topology["name"] = project_name
+        # To avoid unexpected behavior (project start without manual operations just after import)
+        topology["auto_start"] = False
+        topology["auto_open"] = False
+        topology["auto_close"] = False
+        # change node ID
+        node_old_to_new = {}
+        for node in topology["topology"]["nodes"]:
+            new_node_id = str(uuid.uuid4())
+            if "node_id" in node:
+                node_old_to_new[node["node_id"]] = new_node_id
+                _move_node_file(new_project_path, node["node_id"], new_node_id)
```

Comment:
There is an Unresolved reference '_move_node_file' for this line.

Reply:
Sorry, I think someone must change the code, I'll check and re-push it tomorrow morning.

Reply:
Thanks, I have fixed it in https://github.com/GNS3/gns3-server/commit/19bd953d31a9c80414fa53baec54181e934ffefd

Reply:
I am testing it now 👍